documentation-site/components/markdown-elements.js

```diff
@@ -114,9 +113,9 @@ export const DocLink = ({children, href}: {children: string, href: string}) => {
     (parts[0] === '' && parts[1] !== '') || !href.includes('http');
   return (
     <Link href={href}>
-      <StyledLink href={href} {...(internal ? {} : {target: '_blank'})}>
+      <a href={href} {...(internal ? {} : {target: '_blank'})}>
```

Comment:
I removed it, because of two reasons:

* causes `ref` errors
* the bold links look a bit off - I think the links without the heavier weight look better in the body of the main content